wagtail/management/commands/publish_scheduled_pages.py

```diff
@@ -1,115 +1,16 @@
-        dryrun = False
-        if options["dryrun"]:
-            self.stdout.write("Will do a dry run.")
-            dryrun = True
-
-        # 1. get all expired pages with live = True
-        expired_pages = Page.objects.filter(live=True, expire_at__lt=timezone.now())
-        if dryrun:
-            if expired_pages:
-                self.stdout.write("Expired pages to be deactivated:")
-                self.stdout.write("Expiry datetime\t\tSlug\t\tName")
-                self.stdout.write("---------------\t\t----\t\t----")
-                for ep in expired_pages:
-                    self.stdout.write(
-                        "{0}\t{1}\t{2}".format(
-                            ep.expire_at.strftime("%Y-%m-%d %H:%M"), ep.slug, ep.title
-                        )
-                    )
-            else:
-                self.stdout.write("No expired pages to be deactivated found.")
-        else:
-            # Unpublish the expired pages
-            # Cast to list to make sure the query is fully evaluated
-            # before unpublishing anything
-            for page in list(expired_pages):
-                page.unpublish(
-                    set_expired=True, log_action="wagtail.unpublish.scheduled"
-                )
-
-        # 2. get all page revisions for moderation that have been expired
-        expired_revs = [
-            r
-            for r in Revision.page_revisions.filter(submitted_for_moderation=True)
-            if revision_date_expired(r)
-        ]
-        if dryrun:
-            self.stdout.write("---------------------------------")
-            if expired_revs:
-                self.stdout.write(
-                    "Expired revisions to be dropped from moderation queue:"
-                )
-                self.stdout.write("Expiry datetime\t\tSlug\t\tName")
-                self.stdout.write("---------------\t\t----\t\t----")
-                for er in expired_revs:
-                    rev_data = er.content
-                    self.stdout.write(
-                        "{0}\t{1}\t{2}".format(
-                            dateparse.parse_datetime(
-                                rev_data.get("expire_at")
-                            ).strftime("%Y-%m-%d %H:%M"),
-                            rev_data.get("slug"),
-                            rev_data.get("title"),
-                        )
-                    )
-            else:
-                self.stdout.write("No expired revision to be dropped from moderation.")
-        else:
-            for er in expired_revs:
-                er.submitted_for_moderation = False
-                er.save()
-
-        # 3. get all revisions that need to be published
-        revs_for_publishing = Revision.page_revisions.filter(
-            approved_go_live_at__lt=timezone.now()
+        warn(
+            "The `publish_scheduled_pages` management command is deprecated; "
+            "use the `publish_scheduled` command instead.",
```

Comment:
I think we should leave the alias `publish_scheduled_pages` in place indefinitely, and not bother with the deprecation - that way we're not creating extra work for developers who might never use snippet scheduling, and it's not really a big deal to leave this in place in the codebase.

Either way, the documentation needs updating to use the new name.